src/Sarif.Converters/PREFastObjectModel/Defect.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Xml.Serialization;
+
+namespace Microsoft.CodeAnalysis.Sarif.Converters.PREFastObjectModel
+{
+    public class Defect
+    {
+        public SFA SFA { get; set; }
```

Comment:
Why is this here?

Reply:
A PREfast defect has a child node named `SFA`.  I forgot to add `[XmlElement("SFA")]` to this but it seems to deserialize it anyway.

---
In reply to: [146912821](https://github.com/Microsoft/sarif-sdk/pull/657#discussion_r146912821) [](ancestors = 146912821)